server/graphql/v2/input/PaymentMethodInput.ts

```diff
@@ -0,0 +1,37 @@
+import { GraphQLBoolean, GraphQLInputFieldConfigMap, GraphQLInputObjectType, GraphQLString } from 'graphql';
+
+import { PaymentMethodType } from '../enum';
+
+import { CreditCardCreateInput } from './CreditCardCreateInput';
+import { PaypalPaymentDataInput } from './PaypalPaymentDataInput';
+
+export const PaymentMethodInput = new GraphQLInputObjectType({
+  name: 'PaymentMethodInput',
+  description: 'An input to use for creating or retrieving payment methods',
+  fields: (): GraphQLInputFieldConfigMap => ({
+    id: {
+      type: GraphQLString,
+      description: 'The id assigned to the payment method',
+    },
+    type: {
+      type: PaymentMethodType,
+      description: 'Type of this payment method',
+    },
+    name: {
+      type: GraphQLString,
+      description: 'Name of this payment method',
+    },
+    save: {
+      type: GraphQLBoolean,
+      description: 'Wether this payment method should be saved for future payments',
+    },
+    creditCardInfo: {
+      type: CreditCardCreateInput,
+      description: 'When creating a credit card, use this field to set its info',
+    },
+    paypalData: {
```

Comment:
One is `info`, one is `data`, I get their purpose is a bit different but looks strange.

Reply:
I'll rename this one to `paypalInfo` for consistency